docs/contributor/arch/002-fluent-bit-without-file-system-buffer.md

```diff
@@ -0,0 +1,63 @@
+# 1. Fluent Bit without File-System Buffer
+
+Date: 2023-11-23
+
+## Status
+
+Proposed
+
+## Context
+
+The Fluent Bit configuration that we currently generate with the telemetry-manager from a LogPipeline uses a `tail`
+input plugin to read container logs, splits the log stream into multiple pipelines using a `rewrite_tag` filter, applies
+additional pipeline specific filters, and sends the log stream to the configured output. This setup decouples the input
+and outputs so that a blocked output does not affect all other outputs. Consequently, reading logs by the tail plugin is
+never paused. The `rewrite_tag` filter uses a persistent file-system buffer to prevent log loss. The persistent buffer
+is limited to 1 GB to ensure that Kubernetes nodes do not run out of disk space.
+
+The persistent buffer can ensure log loss only for a short period of time. Depending on the amount of generated logs,
+this is usually in the range of minutes to a few hours and might be too short to restore an outage of a log backend or
+detect and solve a faulty configuration.
+
+The removal of the in-cluster Loki backend requires a reevaluation of the pipeline isolation requirements in favor of
```

Comment:
[we don't say "we"](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#pronouns)

```suggestion
After the in-cluster Loki backend has been removed, the pipeline isolation requirements must be reevaluated in favor of pausing the input plugin. The typical setup are clusters with a single LogPipeline or multiple LogPipelines that ingest logs to the same backend; for example, ingesting application logs and Istio access logs into two different indexes of an [SAP Cloud Logging](../../user/integration/sap-cloud-logging) instance. With this typical setup, the benefit of pausing the `tail` input plugin exceeds the pipeline isolation requirements. Kubernetes' [logging architecture](https://kubernetes.io/docs/concepts/cluster-administration/logging/) helps by storing logs persistently on the node file-system and rotating automatically after reaching a certain file size.
```

Kubernetes logging architecture rotates? Or should "rotating" have an object? What is rotated?
Aaah, line 61 says that logs are rotated. Then I'd add a "them":

```suggestion
After the in-cluster Loki backend has been removed, the pipeline isolation requirements must be reevaluated in favor of pausing the input plugin. The typical setup are clusters with a single LogPipeline or multiple LogPipelines that ingest logs to the same backend; for example, ingesting application logs and Istio access logs into two different indexes of an [SAP Cloud Logging](../../user/integration/sap-cloud-logging) instance. With this typical setup, the benefit of pausing the `tail` input plugin exceeds the pipeline isolation requirements. Kubernetes' [logging architecture](https://kubernetes.io/docs/concepts/cluster-administration/logging/) helps by storing logs persistently on the node file-system and rotating them automatically after reaching a certain file size.
```